content/en/monitors/monitor_types/process.md

```diff
@@ -11,74 +11,48 @@ further_reading:
   text: "Schedule a downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
-  text: "Consult your monitor status"
+  text: "Check your monitor status"
 ---
 
 ## Overview
 
-Process monitors are configured to check the status of individual or groups of processes. Process monitors are evaluated every minute.
+Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert you based on the behavior of any group of processes across your hosts or tags.
```

Comment:
"on the behavior" might be a little bit misleading as we only allow users to warn/alert based on the count of processes.